newIDE/app/src/EventsSheet/index.js

```diff
@@ -281,6 +282,12 @@ export default class EventsSheet extends React.Component<Props, State> {
     );
   };
 
+  _escapeSearchPanel = () => {
+    if (this.state.showSearchPanel) {
+      this.setState({ showSearchPanel: false });
+    }
```

Comment:
```suggestion
    this.setState({ showSearchPanel: false });
```

works as well in this case